docs/Usage_tips.md

```diff
@@ -3,7 +3,7 @@
 ## Put BLE functions in a task running on the NimBLE stack core
 
 When commands are sent to the stack from a different core they can experience delays in execution.  
-This library detects this and invokes the esp32 IPC to reroute these commands through the correct core but this also increases overhead.  
+This library is threadsafe. It detects calls from other cores and invokes the esp32 IPC to reroute these commands through the correct core but this also increases overhead.  
```

Comment:
This is no longer relevant, the IPC is no longer used, so just removing this will be best to prevent any confusion.

Reply:
@h2zero ready for re-review!